packages/react-native-skia/modules/Platform/platform.js

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View,
+  Platform,
+  TouchableHighlight
+} from "react-native";
+
+const SimpleViewApp = () => {
```

Comment:
Suggestion from my side: As it is not an App for demonstrating simple View, it will be good to have more relatable name, like PlatformApp or some generic name "App"

Reply:
I think we have to do that from every test app we are developing.

Reply:
@Naveen33-am  Are we retaining the App name as "SimpleViewApp "?